index.html

```diff
@@ -30,6 +30,25 @@
 -->
 
 <div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="col-md-1">
+      </div>
+      <div class="col-md-10 lead text-justify">
+        Interledger is an open protocol suite for sending payments across different <em>ledgers</em>.
```

Comment:
I'm gonna merge it for now but we can definitely keep changing the description later if there are further ways to make it clearer